Client/src/Pages/ForgotPassword/index.jsx

```diff
@@ -5,8 +5,58 @@ import Logomark from "../../assets/Images/key-password.png";
 import EmailTextField from "../../Components/TextFields/Email/EmailTextField";
 import Button from "../../Components/Button";
 import LeftArrow from "../../assets/Images/arrow-left.png";
+import { useState, useEffect } from "react";
+import { recoveryValidation } from "../../Validation/validation";
+import axios from "axios";
+
+const BASE_URL = import.meta.env.VITE_APP_API_BASE_URL;
 
 const ForgotPassword = () => {
+  const [isLoading, setIsLoading] = useState(false); // Used to disable the button while loading so user doesn't submit multiple times
+  const [errors, setErrors] = useState({});
+  const [form, setForm] = useState({
+    email: "",
+  });
+
+  const idMap = {
+    "forgot-password-email-input": "email",
+  };
+
+  useEffect(() => {
+    const { error } = recoveryValidation.validate(form);
+    if (error === undefined) {
+      setErrors({});
+    } else {
+      const validationErrors = error.details.reduce((acc, err) => {
+        return { ...acc, [err.path[0]]: err.message };
+      }, {});
+      setErrors(validationErrors);
+    }
+  }, [form]);
+
+  const handleInput = (e) => {
+    const fieldName = idMap[e.target.id];
+    setForm({ ...form, [fieldName]: e.target.value });
+  };
+
+  const handleSubmit = async () => {
+    //TODO show loading spinner
+    setIsLoading(true);
+    try {
+      const { error } = recoveryValidation.validate(form);
+      if (error !== undefined) {
+        throw error;
+      }
+      const res = await axios.post(`${BASE_URL}/auth/recovery/request`, form);
+      // TODO properly display success
+      alert(JSON.stringify(res.data));
```

Comment:
Instead of showing the alert message, we need to redirect to the `/check-email` route.

Reply:
Done!